src/components/common/card-list.tsx

```diff
@@ -0,0 +1,43 @@
+import { CardClassType } from '@components/card/card';
+import Card from '@models/card';
+import CardComponent from '@components/card/card';
+
+type CardListProps = {
+  cards: Card[];
+  classes: CardClassType;
+  setActiveCardId?: (id: number | undefined) => void;
+};
+
+const Classes = {
```

Comment:
Это не перечисление или enum чтобы делать с большой буквы, на защите могут придраться я бы исправил на 
```suggestion
const сlasses = {
```